xep-0280.xml

```diff
@@ -329,7 +329,7 @@
 
 </section1>
 <section1 topic='Avoiding Carbons for a single message' anchor='avoiding'>
-  <p>Some clients might want to avoid Carbons on a single message, while still keeping all of the other semantics of Carbon support. This might be useful for clients sending end-to-end encrypted messages, for example. To exclude a &MESSAGE; from being forwarded to other Carbons-enabled resources, the sending client add a &lt;private/&gt; element qualified by the namespace "urn:xmpp:carbons:2" as a child content element to the &MESSAGE; stanza.</p>
+  <p>Some clients might want to avoid Carbons on a single message, while still keeping all of the other semantics of Carbon support. This might be useful for clients sending end-to-end encrypted messages, for example. To exclude a &MESSAGE; from being forwarded to other Carbons-enabled resources, the sending client must add a &lt;private/&gt; element qualified by the namespace "urn:xmpp:carbons:2" and a &lt;no-copy/&gt; hint according to &xep0334; as child content elements to the &MESSAGE; stanza.</p>
```

Comment:
If this isn't a "MUST" in the RFC2119 sense (and I agree, I don't think it is), maybe this should be rephrased to prevent implementors from assuming it was a typo in the future then? "the client might add…"?